stepup/README.md

```diff
@@ -117,16 +117,17 @@ STEPUP_VERSION=test
 3. Choose if you want to run the containers in the back- or foreground.
 
 3: Once the containers are up and running, you can run the behat tests
-1. Open a shell in the `behat` container `$ docker exec -it stepup-behat-1 bash`
-2. Run the tests:
+1. Install the required dependencies in the container `$ docker compose exec composer install`
```

Comment:
docker compose exec behat composer install

Reply:
Thanks, changed the command